gradle.properties

```diff
@@ -3,14 +3,14 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/versions.html
-	minecraft_version=1.18
-	yarn_mappings=1.18+build.1
-	loader_version=0.12.6
+	minecraft_version=1.19
+	yarn_mappings=1.19+build.1
+	loader_version=0.14.6
 
 # Mod Properties
 	mod_version = 1.0.1
```

Comment:
Don't forget to increment this!

Reply:
Good idea. Made it ` 1.0.2-Fabric-1.19` in favor of more-sustainable versioning with multiple target Minecraft versions.

Reply:
> Good idea. Made it ` 1.0.2-Fabric-1.19` in favor of more-sustainable versioning with multiple target Minecraft versions.

It already appends the mc version in the build.gradle